### PR TITLE
Update documents

### PR DESCRIPTION
**Pull request by Chuhaa**
## Purpose

- Update documents
- Add comments for client and listener `init`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,389 +1,59 @@
...
-
-- #### [Trigger for deleted event](https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/blob/main/samples/trigger_delete_event.bal)
-
-    This sample shows how to create a trigger on cancelled event. When a new event is cancelled, that event details can be captured in this listener.
+[Google Calendar](https://developers.google.com/calendar) is a time-management and scheduling calendar service developed by Google. It lets users to organize their schedule and share events with others.
+
+This connector provides the capability to programmatically manage events and calendar and listen the events occurred in the calendar. 
+For more information about configuration and operations, go to the module.
+- [ballerinax/googleapis.calendar](https://docs.central.ballerina.io/ballerinax/googleapis.calendar/0.1.9)
+- [ballerinax/googleapis.calendar.listener](https://docs.central.ballerina.io/ballerinax/googleapis.calendar.listener/0.1.9)
```
Comment (LakshanSS) [inline README.md:13]: The latest version of calendar in central is 0.1.8
But in this beta2 release, we are planning to change version to 1.0.0

Comment (Chuhaa) [reply README.md]: Fixed. Thanks

File: README.md [hunk 2]
```diff
@@ -1,389 +1,59 @@
...
-```
-
-4. To build the calendar connector module without the tests:
-```shell script
-    bal build --skip-tests ./calendar
-```
-
+[Google Calendar](https://developers.google.com/calendar) is a time-management and scheduling calendar service developed by Google. It lets users to organize their schedule and share events with others.
+
+This connector provides the capability to programmatically manage events and calendar and listen the events occurred in the calendar.
```
Comment (shaniR) [inline README.md:10]: calendars
and listen to the events

File: README.md [hunk 3]
```diff
@@ -1,389 +1,59 @@
...
-4. To build the calendar connector module without the tests:
-```shell script
-    bal build --skip-tests ./calendar
-```
-
+[Google Calendar](https://developers.google.com/calendar) is a time-management and scheduling calendar service developed by Google. It lets users to organize their schedule and share events with others.
+
+This connector provides the capability to programmatically manage events and calendar and listen the events occurred in the calendar. 
+For more information about configuration and operations, go to the module.
+- [ballerinax/googleapis.calendar](calendar/Module.md)
```
Comment (shaniR) [inline README.md:12]: Let's have a one sentence description about these modules. For e.g. Google Calendar Module.

File: calendar/Module.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
+## Overview
+The module provides the capability to manage events and calendar operations and supports service account authorization that can provide delegated domain-wide access to GSuite domain and support admin to do operations on behalf of the domain users.
```
Comment (shaniR) [inline calendar/Module.md:2]: Shall we rephrase this to: The Google Calendar Connector provides the capability to manage events and calendar operations. It also provides the capability to support service account authorization that can provide delegated domain-wide access to the GSuite domain and support admins to do operations on behalf of the domain users.

File: calendar/Module.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+
+This module supports [Google Calendar API](https://developers.google.com/calendar/api) version V3.
+ 
+## Prerequisites
+Before using this connector in your Ballerina application, complete the following:
+- Create [Google account](https://accounts.google.com/signup/v2/webcreateaccount?utm_source=ga-ob-search&utm_medium=google-account&flowName=GlifWebSignIn&flowEntry=SignUp)
```
Comment (shaniR) [inline calendar/Module.md:8]: Create a Google account

File: calendar/Module.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+- Obtain tokens - Follow [this link](https://developers.google.com/identity/protocols/oauth2)
```
Comment (shaniR) [inline calendar/Module.md:9]: Obtain Tokens
Follow the steps [here](https://developers.google.com/identity/protocols/oauth2) to obtain tokens.

File: calendar/Module.md [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+
+## Quickstart
+To use the Google Calendar connector in your Ballerina application, update the .bal file as follows:
+
+### Step 1: Import connector
+First, import the `ballerinax/googleapis.calendar` module into the Ballerina project.
```
Comment (shaniR) [inline calendar/Module.md:15]: Let's remove the word "First,"

File: calendar/Module.md [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+```ballerina
+import ballerinax/googleapis.calendar;
+```
+
+### Step 2: Create a new connector instance
+Create a `calendar:CalendarConfiguration` with the OAuth2 tokens obtained, and initialize the connector with it.
```
Comment (shaniR) [inline calendar/Module.md:21]: OAuth2.0

Comment (shaniR) [reply calendar/Module.md:21]: obtained and initialize

File: calendar/Module.md [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+            },
+            summary: "Sample Event"
+        };
+
+    public function main() returns error? {
+        calendar:Event response = check calendarClient->createEvent(calendarId, event);
+    }
+    ``` 
+
+- Create a quick event using service account
```
Comment (shaniR) [inline calendar/Module.md:70]: using a service account

File: calendar/Package.md [hunk 10]
```diff
@@ -1,349 +1,20 @@
...
 
-# Ballerina Google Calendar Connector 
-Connects to Google Calendar using Ballerina.
+## Package overview
+The `ballerinax/googleapis.calendar` is a [Ballerina](https://ballerina.io/) connector for Google Calendar.
 
-# Introduction
-## Google Calendar
-[Google Calendar](https://developers.google.com/calendar) is a time-management and scheduling calendar service developed by Google. It lets users to organize their schedule and share events with others. The Google Calendar endpoint allows you to access the Google Calendar API Version v3 through Ballerina.
+This package provides the capability to manage calendars and events in the Google Calendar and listen the events occurred in the calendar.
```
Comment (shaniR) [inline calendar/Package.md:6]: listen to events

**Review by shaniR — CHANGES_REQUESTED** [review]

File: calendar/Module.md (hunk 8 above)
Comment (Chuhaa) [reply calendar/Module.md]: Fixed in https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/pull/45/commits/33c24844a5693ce13fa4854db3a209ae2fcfb884

File: calendar/Module.md (hunk 7 above)
Comment (Chuhaa) [reply calendar/Module.md]: Fixed in https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/pull/45/commits/33c24844a5693ce13fa4854db3a209ae2fcfb884

File: calendar/Module.md (hunk 5 above)
Comment (Chuhaa) [reply calendar/Module.md]: Fixed in https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/pull/45/commits/33c24844a5693ce13fa4854db3a209ae2fcfb884

File: calendar/Module.md (hunk 4 above)
Comment (Chuhaa) [reply calendar/Module.md]: Fixed in https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/pull/45/commits/33c24844a5693ce13fa4854db3a209ae2fcfb884

File: README.md (hunk 3 above)
Comment (Chuhaa) [reply README.md]: Added in https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/pull/45/commits/33c24844a5693ce13fa4854db3a209ae2fcfb884

File: README.md (hunk 2 above)
Comment (Chuhaa) [reply README.md]: Fixed in https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/pull/45/commits/33c24844a5693ce13fa4854db3a209ae2fcfb884

File: calendar/Module.md (hunk 6 above)
Comment (Chuhaa) [reply calendar/Module.md]: Fixed in https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/pull/45/commits/33c24844a5693ce13fa4854db3a209ae2fcfb884

File: calendar/Module.md (hunk 9 above)
Comment (Chuhaa) [reply calendar/Module.md]: Decided to remove quick reference.

File: calendar/Package.md (hunk 10 above)
Comment (Chuhaa) [reply calendar/Package.md]: Fixed in https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/pull/45/commits/33c24844a5693ce13fa4854db3a209ae2fcfb884

**Review by abeykoon — APPROVED** [review]

File: calendar/endpoints.bal [hunk 11]
```diff
@@ -17,14 +17,22 @@
 import ballerina/http;
 import ballerina/jwt;
 
-# Client for Google Calendar connector.
+# Ballerina Google Calendar connector provides the capability to access Google Calendar API.
+# The connector let you perform calendar and event management operations.
```
Comment (shaniR) [inline calendar/endpoints.bal:21]: lets you perform

File: calendar/endpoints.bal [hunk 12]
```diff
@@ -17,14 +17,22 @@
...
 # + calendarClient - HTTP client endpoint
 @display {label: "Google Calendar", iconPath: "logo.png"}
-public client class Client {
-    public http:Client calendarClient;
-    private ClientOAuth2ExtensionGrantHandler clientHandler;
+public isolated client class Client {
+    private final http:Client calendarClient;
+    private final ClientOAuth2ExtensionGrantHandler clientHandler;
 
+    # Initializes the connector. During initialization you can pass either http:BearerTokenConfig if you have a bearer
```
Comment (shaniR) [inline calendar/endpoints.bal:29]: To initialize, you can pass either the http:BearerTokenConfig if you have a bearer

File: calendar/endpoints.bal [hunk 13]
```diff
@@ -17,14 +17,22 @@
...
+    # token or http:OAuth2RefreshTokenGrantConfig if you have Oauth tokens.
```
Comment (shaniR) [inline calendar/endpoints.bal:30]: token or the http:OAuth2RefreshTokenGrantConfig if you have OAuth 2.0 tokens.
